docs/source/index.md

```diff
@@ -37,8 +37,8 @@ Performance reports for the provided samples are hosted on the [documentation pa
     *make sure that the version/tag of the benchmark repository matches the version of Caliper you are using! For example, if you are using Caliper v0.6.0, then `checkout` the `v0.6.0` tag after cloning the benchmark repository. The `main` branch of the benchmark repository corresponds to the latest `unstable` Caliper version.*

```

Comment:
Under Sample Networks just above on line 33
```
Performance reports for the provided samples are hosted on the [documentation pages of the repository](https://hyperledger.github.io/caliper-benchmarks/).
```
Should be removed as these don't exist anymore